packages-exp/auth-exp/src/model/auth.ts

```diff
@@ -20,45 +20,84 @@ import * as externs from '@firebase/auth-types-exp';
 import { PopupRedirectResolver } from './popup_redirect';
 import { User } from './user';
 
+/** @internal */
 export type AppName = string;
+/** @internal */
 export type ApiKey = string;
+/** @internal */
 export type AuthDomain = string;
 
+/** {@inheritdoc @firebase/auth-types#Config} */
 export interface ConfigInternal extends externs.Config {
+  /**
+   * @internal
+   * @readonly
+   */
   emulator?: {
     url: string;
   };
 }
 
+/** {@inheritdoc @firebase/auth-types#Auth} */
 export interface Auth extends externs.Auth {
+  /**
+   * {@inheritdoc @firebase/auth-types#Auth.currentUser}
+   * @readonly
+   */
   currentUser: externs.User | null;
+  /** @internal */
   _canInitEmulator: boolean;
+  /** @internal */
   _isInitialized: boolean;
+  /** @internal */
   _initializationPromise: Promise<void> | null;
+  /** {@inheritdoc @firebase/auth-types#Auth.updateCurrentUser} */
```

Comment:
this should be `@internal` still. The public `updateCurrentUser()` is in the public types